### DISCUSSION TITLE
Front-end lint errors in the console on local

### DISCUSSION THREAD

**Discussion by srgiola** [awslabs/performance-dashboard-on-aws#750] [Q&A]
Dear developers.

Apart from creating an env.js file with the previous parameters, what else is necessary to connect the frontend locally to the backend that is already in the cloud?

I'm getting multiples errror each time that i run locally the front-end at the same time it's not properly working as the version that is already on the cloud.

Errors for example: 

```
Failed to load resource: the server responded with a status of 400 ()

Access to XMLHttpRequest at 'https://performancedash-release-#####-us-east-1-datasets.s3.us-west-1.amazonaws.com/public/logo/fac####.png?x-id=GetObject' from origin 'http://localhost:3000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
        
[ERROR] 25:58.367 axios-http-handler Error: Network Error
at createError (createError.js:16)
at XMLHttpRequest.handleError (xhr.js:84)
```

Thanks in advance.

**Comment by trietlu** [comment]
Hi @srgiola,

I assume you're trying to run the front end locally for development/debugging?

**Reply by srgiola** [reply]
Yes, that's what i'm trying to do.

The most exact problem i think is that it is not able to get all the data stored from the widget and the S3, but i'm not sure.

**Comment by trietlu** [comment]
Hi @srgiola,

Did you follow the instructions in the [frontend README](https://github.com/awslabs/performance-dashboard-on-aws/tree/mainline/frontend)?  What are you seeing differently when running the frontend locally versus when running the frontend hosted on AWS?

**Reply by DenisVillalta** [reply]
Hello, I am a MyappSoftware contributor and following up on your comments, I followed the instructions in the README file and if I get the site up but not complete, it is very slow and when I open the browser console it shows 5 errors. 
I leave a screenshot, and I am waiting for your comments and help.
![Error Dashboard](https://user-images.githubusercontent.com/64240235/149976586-4126faf9-db54-45f3-8f31-1fa20b8788b8.PNG)

**Comment by trietlu** [comment]
@DenisVillalta To help you troubleshoot, we'll need to see your public/env.js file.  If you're with MyAppSoftware, I mentioned to your management that you can contact us on Chime for a quicker turnaround

**Reply by DenisVillalta** [reply]
Thank you for your help, I believe we have already coordinated a Chime call for next week so we can follow up on this issue on that call.

**Comment by kharbutli-meta** [comment]
Hello @srgiola, to help you debug this issue, can you provide answers to the following:
1. Did you confirm that your backend is up and running? How did you confirm that?
2. Can you provide the contents of your frontend/public/env.js file?

Thanks,
